src/run/run.c

```diff
@@ -2576,10 +2576,12 @@ static int run(int argc, char* argv[]) {
 
                 _cleanup_free_ char *command = NULL;
                 r = find_executable(arg_cmdline[0], &command);
-                if (r < 0)
+                if (ERRNO_IS_NEG_PRIVILEGE(r))
```

Comment:
Hmm, this seems to be too relaxed too much? We should insist on at least one user having x access?

I think find_executable_full should gain a new flag for this.

Reply:
dunno. strictly speaking the call is entirely redundant. The server side executes the same thing after all. The only reason why we do it, is to resolve this in caller context to make thing more stable and easier to debug.

hence, if in doubt we should just skip this, which this thing now does.